nixos/modules/services/misc/mediatomb.nix

```diff
@@ -0,0 +1,260 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  uid = config.ids.uids.mediatomb;
+  gid = config.ids.gids.mediatomb;
+  cfg = config.services.mediatomb;
+
+  mtConf = pkgs.writeText "config.xml" ''
```

Comment:
mediatomb will normally generate this file on first run. An alternative approach would be to trigger mediatomb once on build + mod the resultant file, but this seems more nix-ish?